.editorconfig

```diff
@@ -37,7 +37,7 @@ indent_size = 2
 
 # Dotnet diagnostic settings
 [*.{cs,vb}]
-dotnet_diagnostic.xUnit2018.severity = suppress # "do not compare an object's exact type to the abstract class" is a valid assert, but very noisy right now
```

Comment:
I assume this fixes a bunch of spurious warnings in the build?

Reply:
Yeah, this is reacting to a change where we renamed "suppress" in editorconfig to "none"